examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -15,181 +14,162 @@ class CupertinoAlertDemo extends StatefulWidget {
 }
 
 class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
-  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
-
   void showDemoDialog<T>({BuildContext context, Widget child}) {
-    showDialog<T>(
+    showCupertinoDialog<T>(
       context: context,
-      barrierDismissible: false,
       builder: (BuildContext context) => child,
-    ).then<void>((T value) {
```

Comment:
It's a shame to punt the code that shows how to process the value "returned" by a dialog.  The rest of the demo specifies String for T and Navigator.pop provides a value ... Maybe just show the returned value in field at the bottom of the page?

Reply:
Indeed. Added it back as you suggested with plain text at the bottom.